pkg/runctl/cfg/config.go

```diff
@@ -206,6 +214,15 @@ func processMainConfig(configData map[string]string, dest *PipelineRunsConfigStr
 		return err
 	}
 
+	if dest.TimeoutWait, err =
+		parseDuration(mainConfigKeyTimeoutWait); err != nil {
+		return err
+	}
+
+	if isZeroDuration(dest.TimeoutWait) {
+		dest.TimeoutWait = metav1Duration(time.Duration(defaultWaitTimeoutMinutes * time.Minute))
+	}
```

Comment:
I think this package is not supposed to apply default values. It just provides the mechanics to load the config.

The default value should be applied by the run controller.

Reply:
See https://github.com/SAP/stewardci-core/pull/345/commits/a842b47996c3d360b2da4ec16ba0344d43efee16